ClientRuntimes/NodeJS/lib/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "ms-rest",
+  "author": {
+    "name": "Microsoft Corporation",
+    "email": "azsdkteam@microsoft.com",
+    "url": "https://github.com/Azure/AutoRest"
+  },
+  "version": "0.1.1",
+  "description": "Client Runtime for nodeJS client libraries generated using AutoRest",
+  "tags": ["node", "microsoft", "autorest", "clientruntime"],
+  "keywords": ["node", "microsoft", "autorest", "clientruntime"],
+  "main": "./lib/msRest.js",
+  "engines": {
+    "node": ">= 0.6.15"
```

Comment:
Again, do we really work with 0.6.15?  Can we change this in a big way for the new runtime?